### PR TITLE
change Monasca documentation (SCRD-7786)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -5403,23 +5445,23 @@ sudo openstack router add subnet <replaceable>router2</replaceable> priv-net-sub
...
-   </varlistentry>
   </variablelist>
 
   <bridgehead renderas="sect3"><guimenu>kafka: tunables for
 Kafka</guimenu>
   </bridgehead>
 
   <variablelist>
+   <varlistentry>
+    <term>config_options</term>
```
Comment (witekest) [inline xml/depl_nodes.xml]: Also here the parameter hierarchy is flattened. `replicas`, `partitions` and `config_options` are all options of `topics` list.
I think it's clear enough for the user though when she looks at the template https://github.com/crowbar/crowbar-openstack/blob/master/chef/data_bags/crowbar/template-monasca.json

File: xml/depl_nodes.xml [hunk 2]
```diff
@@ -5438,7 +5480,30 @@ Kafka</guimenu>
      </para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+    <term>partitions</term>
+    <listitem>
+     <para>
+      Controls how many logs the topic will be sharded into
```
Comment (asettle) [inline xml/depl_nodes.xml]:
``` suggestion
Controls the amount of logs the topic shards into
```

File: xml/depl_nodes.xml [hunk 3]
```diff
@@ -5438,7 +5480,30 @@ Kafka</guimenu>
...
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>replicas</term>
+    <listitem>
+     <para>
+      Controls how many servers will replicate each message that is written
```
Comment (asettle) [inline xml/depl_nodes.xml]:
``` suggestion
Controls how many servers replicate the messages that are written
```

Are these error messages or what exactly? Would be helpful to define

File: xml/depl_nodes.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/depl_nodes.xml]: thank you for the link. makes things clearer.

changes made according to comments.

File: xml/depl_nodes.xml [hunk 4]
```diff
@@ -5438,72 +5472,142 @@ Kafka</guimenu>
...
-  </bridgehead>
-
-  <variablelist>
    <varlistentry>
-    <term>influxdb_retention_policy</term>
+    <term>topics</term>
     <listitem>
      <para>
-      Number of days to keep metrics records in influxdb.
+      Kafka settings
```
Comment (witekest) [inline xml/depl_nodes.xml]: "List of topics" would be better I guess.

File: xml/depl_nodes.xml [hunk 5]
```diff
@@ -5438,72 +5472,142 @@ Kafka</guimenu>
...
+       <para>
+        transformed-log
+       </para>
+      </listitem>
+     </itemizedlist>
      <para>
-      For an overview of all supported values, see
-      <link
-         xlink:href="https://docs.influxdata.com/influxdb/v1.1/query_language/database_management/#create-retention-policies-with-create-retention-policy"></link>.
+      The following are options of the topic list:
```
Comment (witekest) [inline xml/depl_nodes.xml]: The following are options of every topic.

**Pull request merged**